plugin-server/src/worker/ingestion/process-event.ts

```diff
@@ -798,4 +792,62 @@ export class EventsProcessor {
             )
         }
     }
+
+    private async handleTablesDependingOnPersonID(
```

Comment:
Note: This should be a separate PR given the risk of accidentally breaking ingestion.